src/components/Editor/Runners/PythonRunner/SkulptRunner/SkulptRunner.jsx

```diff
@@ -304,8 +304,10 @@ const SkulptRunner = ({ active, outputPanels = ["text", "visual"] }) => {
       const fileName = err.traceback[0].filename.replace(/^\.\//, "");
 
       if (errorType === "ImportError" && window.crossOriginIsolated) {
+        const articleLink =
+          "https://help.editor.raspberrypi.org/hc/en-us/articles/30841379339924-What-Python-libraries-are-available-in-the-Code-Editor";
```

Comment:
This is probably fine for now, but I'm wondering whether in the future we might want to link this off to different locales with translations of the article 🤔

Reply:
Thanks @loiswells97 for you review!!
Locales were added

Reply:
This was just a thought for the future sorry, not something we want right now... I just checked and it looks like it's only the `en-us` locale that exists on the help site for now, `en-GB` for example gives a 404